src/admin/src/components/page/advisoryDashboard/AdvisoryDashboard.js

```diff
@@ -67,99 +71,113 @@ export default function AdvisoryDashboard({
       cmsAxios.get(
         `public-advisory-audits?_limit=1500&_sort=advisoryDate:DESC${parkIdQuery}`,
         {
           headers: { Authorization: `Bearer ${keycloak.token}` },
         }
       ),
-    ]);
-
-    const managementAreas = response[0];
-    const publicAdvisories = response[1].data;
-
-    const getCurrentPublishedAdvisories = (cmsData, setCmsData) => {
-      const advisoryStatuses = getAdvisoryStatuses(cmsData, setCmsData);
-      if (advisoryStatuses) {
-        const publishedStatus = advisoryStatuses.filter(
-          (as) => as.code === "PUB"
-        );
-        if (publishedStatus && publishedStatus[0]) {
-          cmsAxios
-            .get(
-              `/public-advisories?_advisoryStatus=${publishedStatus[0].id}&_limit=-1`
-            )
-            .then((res) => {
-              const result = res.data;
-              let publishedAdvisories = [];
-              result.forEach((ad) => {
-                publishedAdvisories = [
-                  ...publishedAdvisories,
-                  ad.advisoryNumber,
-                ];
-              });
-              if (isMounted.current) {
-                setPublishedAdvisories([...publishedAdvisories]);
-              }
-            });
-        }
-      }
-    };
+    ])
+    .then((response) => {
+      getCurrentPublishedAdvisories(cmsData, setCmsData);
 
-    getCurrentPublishedAdvisories(cmsData, setCmsData);
+      const managementAreas = response[0];
+      const publicAdvisories = response[1].data;
 
-    const regionParksCount = managementAreas.reduce((region, item) => {
-      region[item.region.id] =
-        (region[item.region.id] || 0) + item.protectedAreas.length;
-      return region;
-    }, {});
+      const regionParksCount = managementAreas.reduce((region, item) => {
+        region[item.region.id] =
+          (region[item.region.id] || 0) + item.protectedAreas.length;
+        return region;
+      }, {});
 
-    const data = publicAdvisories.map((publicAdvisory) => {
-      publicAdvisory.expired = publicAdvisory.expiryDate < today ? "Y" : "N";
-      publicAdvisory.associatedParks =
-        publicAdvisory.protectedAreas
-          .map((p) => p.protectedAreaName)
-          .join(", ") +
-        publicAdvisory.regions.map((r) => r.regionName).join(", ");
-
-      let regionsWithParkCount = [];
-      if (publicAdvisory.regions.length > 0) {
-        publicAdvisory.regions.forEach((region) => {
-          region.count = regionParksCount[region.id];
-          regionsWithParkCount = [...regionsWithParkCount, region];
-        });
-        publicAdvisory.regions = regionsWithParkCount;
+      const updatedPublicAdvisories = publicAdvisories.map((publicAdvisory) => {
+        publicAdvisory.expired = publicAdvisory.expiryDate < today ? "Y" : "N";
+        publicAdvisory.associatedParks =
+          publicAdvisory.protectedAreas
+            .map((p) => p.protectedAreaName)
+            .join(", ") +
```

Comment:
Either One line/shorten the height

Reply:
Single line goes beyond 80 characters in one line hence it was written like that.

Reply:
done!